CHANGELOG.md

```diff
@@ -11,30 +11,36 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
+
+## [1.21.0/0.46.0/0.15.0/0.1.0] - 2023-11-10
```

Comment:
What is the `v0.1.0` here?

Reply:
The new `go.opentelemetry.io/contrib/config` module.